lms/util/via.py

```diff
@@ -0,0 +1,43 @@
+from urllib import parse
+
+from pyramid.httpexceptions import HTTPBadRequest
+
+
+def via_url(request, document_url):
+    parts = parse.urlparse(document_url)
+    query_string = parts.query
+
+    if _auto_provisioning_feature_enabled(request):
```

Comment:
Would we want to open the sidebar automatically regardless of whether or not auto-provisioning is enabled?